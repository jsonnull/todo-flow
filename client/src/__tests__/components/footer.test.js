import React from 'react';
import renderer from 'react-test-renderer';
import { createStore } from 'redux';
import { Provider } from 'react-redux';

import reducer from '../../reducers';
import Footer from '../../components/footer';

describe('Footer', () => {
  let tree;
  it('renders without crashing', () => {
    const store = createStore(reducer);
    const component = renderer.create(
      <Provider store={store}><Footer /></Provider>
    );
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
