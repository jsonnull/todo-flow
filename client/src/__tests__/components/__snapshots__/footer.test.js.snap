exports[`Footer renders without crashing 1`] = `
<p>
  Show:
   
  <span>
    All
  </span>
  , 
  <a
    href="#"
    onClick={[Function]}>
    Active
  </a>
  , 
  <a
    href="#"
    onClick={[Function]}>
    Completed
  </a>
</p>
`;
